mix.exs

```diff
@@ -37,9 +35,12 @@ defmodule Ecto.Mixfile do
 
   defp deps do
     [{:poolboy, "~> 1.5"},
-     {:sbroker, "~> 0.7", optional: true},
+     {:sbroker, "~> 0.7"},
      {:decimal, "~> 1.0"},
-     {:postgrex, "~> 0.10", optional: true},
+     {:connection, "~> 1.0.2", override: true},
+     {:backoff, "1.1.1", override: true},
```

Comment:
We can remove connection when mariaex gets db_connection support.
I will remove backoff tomorrow.